source/unwind.js

```diff
@@ -0,0 +1,42 @@
+import _curry2 from './internal/_curry2.js';
+import _isArray from './internal/_isArray.js';
+import _map from './internal/_map.js';
+import _assoc from './internal/_assoc.js';
+
+/**
+ *
+ * Deconstructs an array field from the input documents to output a document for each element.
+ * Each output document is the input document with the value of the array field replaced by the element.
+ *
+ * @func
+ * @memberOf R
+ * @category Object
+ * @sig String -> {k: [v]} -> [{k: v}]
+ * @param {String} key The key to determine which property of the object should be unwind
+ * @param {Object} object The object containing list under property named as key which is to unwind
+ * @return {List} A new list of object containing the value of input key having list replaced by each element in the object.
+ * @example
+ *
+ *  R.unwind('hobbies', {
```

Comment:
I don't know if this is done automatically by the documentation website, but if not, can we clean the example spaces? Line 20 has an extra space, like 21,22, 23 needs an extra space -- line 26 and 27 have way too much space.